Sources/TableViewDriver.swift

```diff
@@ -99,57 +97,16 @@ open class TableViewDriver: NSObject {
     // MARK: Change and UI Update Handling
 
     /// Updates all currently visible cells and sections, such that they reflect the latest
-    /// state decribed in their respective view models. Typically this method should not be
-    /// called directly, as it is called automatically whenever the `tableViewModel` property
-    /// is updated.
-    public func refreshViews(refreshContext: TableRefreshContext = .unknown) {
-        guard let sections = self.tableViewModel?.sectionModels, !sections.isEmpty else {
-            return
-        }
-
-        let visibleIndexPaths = self.tableView.indexPathsForVisibleRows ?? []
-
-        // Collect the index paths and views models to reload
-        let indexPathsAndViewModelsToReload = visibleIndexPaths.compactMap { indexPath in
```

Comment:
All of the visible row/section business is now handled by just reloading the table.